project/AddMetaInfLicenseFiles.scala

```diff
@@ -0,0 +1,253 @@
+      |      unless required by applicable law (such as deliberate and grossly
+      |      negligent acts) or agreed to in writing, shall any Contributor be
+      |      liable to You for damages, including any direct, indirect, special,
+      |      incidental, or consequential damages of any character arising as a
+      |      result of this License or out of the use or inability to use the
+      |      Work (including but not limited to damages for loss of goodwill,
+      |      work stoppage, computer failure or malfunction, or any and all
+      |      other commercial damages or losses), even if such Contributor
+      |      has been advised of the possibility of such damages.
+      |
+      |   9. Accepting Warranty or Additional Liability. While redistributing
+      |      the Work or Derivative Works thereof, You may choose to offer,
+      |      and charge a fee for, acceptance of support, warranty, indemnity,
+      |      or other liability obligations and/or rights consistent with this
+      |      License. However, in accepting such obligations, You may act only
+      |      on Your own behalf and on Your sole responsibility, not on behalf
+      |      of any other Contributor, and only if You agree to indemnify,
+      |      defend, and hold each Contributor harmless for any liability
+      |      incurred by, or claims asserted against, such Contributor by reason
+      |      of your accepting any such warranty or additional liability.
+      |
+      |   END OF TERMS AND CONDITIONS
+      |
+      |   APPENDIX: How to apply the Apache License to your work.
+      |
+      |      To apply the Apache License to your work, attach the following
+      |      boilerplate notice, with the fields enclosed by brackets "[]"
+      |      replaced with your own identifying information. (Don't include
+      |      the brackets!)  The text should be enclosed in the appropriate
+      |      comment syntax for the file format. We also recommend that a
+      |      file or class name and description of purpose be included on the
+      |      same "printed page" as the copyright notice for easier
+      |      identification within third-party archives.
+      |
+      |   Copyright [yyyy] [name of copyright owner]
+      |
+      |   Licensed under the Apache License, Version 2.0 (the "License");
+      |   you may not use this file except in compliance with the License.
+      |   You may obtain a copy of the License at
+      |
+      |       http://www.apache.org/licenses/LICENSE-2.0
+      |
+      |   Unless required by applicable law or agreed to in writing, software
+      |   distributed under the License is distributed on an "AS IS" BASIS,
+      |   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      |   See the License for the specific language governing permissions and
+      |   limitations under the License.
+      |""".stripMargin
+
+  val notice =
+    """
+      |Apache Celeborn
```

Comment:
Refer to Apache Pekko, it also uses `Apache Pekko` instead of `Apache Pekko (incubating)`. cc @pan3793 

https://github.com/apache/incubator-pekko/blob/c78e2d76102ee0f9768ce7bca4bbbd0f71a5e9a1/legal/PekkoNotice.txt#L1-L2